components/screens/WishList.js

```diff
@@ -1,13 +1,30 @@
 import React from 'react';
+import { Button } from 'react-native';
 import { View, ScrollView, Text, StyleSheet } from 'react-native';
 import NavBar from '../NavBar';
 
-function WishList({navigation}) {
+function WishList({ screenProps,navigation }) {
+
+    const removeWishListItem = () =>{
+        alert('Item Removed')
+
+        const updatedList = screenProps.wishList;
+
+        for (let index=0;index<updatedList.length;index++){
+            if (updatedList[index].id == 0){
+                updatedList.splice(index,1)
+            };
+        };
+        screenProps.wishListButtonPress(updatedList);
+    }
+
     return (
         <View style={styles.container}>
             <ScrollView>
                 <View style={styles.wishlist_container}>
                     <Text>Wish List</Text>
+                    <Text>{screenProps.wishList[0].name}</Text>
```

Comment:
can you check if the `wishList` is array consisting of the 0th element before checking `name` on it.